setup.py

```diff
@@ -37,11 +37,9 @@
     # TODO: remove pyopencl when dpctl exposes preferred_work_group_size_multiple
     # see https://github.com/IntelPython/dpctl/issues/886
     install_requires=["scikit-learn", "numba-dpex", "dpctl", "pyopencl"],
-    extra_requires=[dict(benchmark=["scikit-learn-intelex, pandas"])],
+    extra_require=[dict(benchmark=["scikit-learn-intelex, pandas"])],
```

Comment:
```suggestion
    extras_require=[dict(benchmark=["scikit-learn-intelex, pandas"])],
```
apparently there's an "s" on extra